cfme/common/provider.py

```diff
@@ -2,19 +2,18 @@
 from functools import partial
 
 from manageiq_client.api import APIException
+from widgetastic.widget import View, Text
+from widgetastic_patternfly import Input, Button
 
-from cfme.base.credential import Credential, EventsCredential, TokenCredential, SSHCredential, \
-    CANDUCredential, AzureCredential, ServiceAccountCredential
+from cfme.base.credential import (Credential, EventsCredential, TokenCredential, SSHCredential,
```

Comment:
match style of existing large import, start the modules on a new line, indented. (cfme.exceptions import)